### PR TITLE
[PORT][NON-MODULAR] Physical Age & Chronological Age

### PR DESCRIPTION
**Pull request by theselfish**
See https://github.com/NovaSector/NovaSector/pull/1161 for details.

Only change was upping the chronological age because 400 seemed a tad arbitrary.

:cl: A.C.M.O.
qol: Changed maximum age to 100. Renamed Age field to Physical Age.
add: Added a new character preferences field for Chronological Age with a maximum of 400.
add: Added Chronological Age field to medical and security records computers.
add: Added fields for Physical Age and Chronological age to the Medical Records app for modular computers.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/records/medical.dm [hunk 1]
```diff
@@ -80,6 +81,7 @@
 	var/list/data = list()
 	data["min_age"] = AGE_MIN
 	data["max_age"] = AGE_MAX
+	data["max_chrono_age"] = AGE_CHRONO_MAX // NOVA EDIT ADDITION - Chronological age
```
Comment (out-of-phaze) [inline code/game/machinery/computer/records/medical.dm]: still says Nova

File: code/__DEFINES/mobs.dm [hunk 2]
```diff
@@ -440,7 +440,8 @@
 //MINOR TWEAKS/MISC
 //#define AGE_MIN 17	//youngest a character can be //ORIGINAL
 #define AGE_MIN	18	//youngest a character can be //SKYRAT EDIT CHANGE - age
-#define AGE_MAX 85 //oldest a character can be
+#define AGE_MAX 100 //oldest a character can be //SKYRAT EDIT CHANGE - Increase max character age to 100 - ORIGINAL: #define AGE_MAX 85 //oldest a character can be
+#define AGE_CHRONO_MAX 9999 //SKYRAT EDIT ADDITION - Chronological age
```
Comment (out-of-phaze) [inline code/__DEFINES/mobs.dm:444]: pr body says max for this is 400 but it appears to be 9999 instead

Comment (theselfish) [reply code/__DEFINES/mobs.dm:444]: Changed that, forgot to mention it. 400 seemed low and kinda arbitrary.

File: code/game/machinery/computer/records/medical.dm (hunk 1 above)
Comment (theselfish) [inline code/game/machinery/computer/records/medical.dm]:
```suggestion
	data["max_chrono_age"] = AGE_CHRONO_MAX // SKYRAT EDIT ADDITION - Chronological age
```

**Pull request merged**